operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md

```diff
@@ -33,16 +34,14 @@ The management of keys on NCNs is achieved by the `trust-csm-ssh-keys` and
 `passwordless-ssh` Ansible roles in the CSM configuration management repository.
 The SSH keypair is applied to management nodes via NCN personalization.
 
-Choose one option from the following sections to enable or disable passwordless
-SSH on NCNs.
 
-### Option 1: Use the CSM-provided SSH Keys
+### Use the CSM-provided SSH Keys
```

Comment:
Why the removal of the "Option #" in the subsection headers? TechPubs requested them to delineate that that only one of these needs to be chosen.